src/main/kotlin/blackject/model/Participant.kt

```diff
@@ -4,17 +4,25 @@ package blackject.model
  * 참가자들 관리 클래스
  * */
 class Participant(
+    val dealer: Dealer,
     val persons: List<Person>
 ) {
 
+    fun getAllPerson(): List<Person> = persons.plus(dealer)
+
     companion object {
         private const val DELIMITER_NAME = ","
         fun addPerson(name: String?): Participant {
             require(!name.isNullOrEmpty())
-            val list = name
-                .split(DELIMITER_NAME)
-                .map { Person(name = it) }
-            return Participant(list)
+            val list = mutableListOf<Person>().apply {
+                addAll(
+                    name
+                        .split(DELIMITER_NAME)
+                        .map { Person(type = PersonType.NORMAL, name = it) }
+                )
+            }
+
+            return Participant(Dealer(), list)
```

Comment:
```suggestion
            return name.split(DELIMITER_NAME)
                .map { Person(type = PersonType.NORMAL, name = it) }
                .let { Participant(Dealer(), it) }
```
kotlin의 함수형 패러다임을 이용한다면 이렇게도 변경할 수 있습니다 🙂